tests/apps/mip/conftest.py

```diff
@@ -133,7 +133,9 @@ def tb_api():
 @pytest.fixture(scope="session")
 def mip_api():
     """MipAPI fixture"""
-    _mip_api = MipAPI(script="test/fake_mip.pl", pipeline="analyse rd_dna")
+    _mip_api = MipAPI(
+        script="test/fake_mip.pl", pipeline="analyse rd_dna", conda_env="S_mip8.2_rd-rna"
```

Comment:
```suggestion
        script="test/fake_mip.pl", pipeline="analyse rd_dna", conda_env="S_mip_rd-rna"
```

more general is better I think unless the specifics are needed somehow, ditto in the following occurances

Reply:
Good suggestion. Fixed